nncf/api/compression.py

```diff
@@ -20,6 +21,28 @@
 ModelType = TypeVar('ModelType')
 
 
+class Statistics(ABC):
+    """
+    Contains a data collection and provides a way for its human-readable representation.
+    """
+
+    @abstractmethod
+    def as_str(self) -> str:
+        """
+        Returns a representation of the statistics as a human-readable string.
+
+        :return: A representation of the statistics as a human-readable string.
+        """
+
+    @abstractmethod
+    def as_dict(self) -> Dict[str, Any]:
+        """
+        Returns a representation of the statistics as built-in data types.
+
+        :return: A representation of the statistics as built-in data types.
```

Comment:
Should add a note on what the string keys in the dict represent.

Reply:
The `as_dict()` method was removed after refactoring. For this reason, your comment is not applicable now.